src/main/java/bio/terra/workspace/db/ResourceDao.java

```diff
@@ -324,6 +324,21 @@ public void createControlledResource(ControlledResource controlledResource)
           "No cloud context found in which to create a controlled resource");
     }
 
+    // scan entire attributes column for existing controlled resource
+    // TODO: use a dedicated URI column to make this more efficient PF-690
+    final String attributesSql =
+        String.format(
+            "SELECT attributes FROM resource "
+                + "WHERE resource_type = '%s' "
+                + "AND stewardship_type = 'CONTROLLED'",
```

Comment:
In general, we have been using parameters to substitute values in queries.

Suggest populating the stewardship_type with StewardshipType.CONTROLLED.toSql() so that you are not relying on the specific string in the query.